schemas/documents/shared/project.js

```diff
@@ -25,6 +25,16 @@ export default {
         },
       ],
     },
+    {
```

Comment:
@clevinson see my comment in https://regennetwork.atlassian.net/browse/APP-266?focusedCommentId=10687

Reply:
Sounds good. Thanks! I hadn't thought of that edge case.